src/Account.java

```diff
@@ -0,0 +1,52 @@
+/**
+ * Created by ken-linux2 on 12/07/16.
+ */
+
+public class Account {
+
+    private String account_name;
```

Comment:
Please use camel case in field - accountName. Here name is enough as we are in account.